mysql/const.go

```diff
@@ -55,6 +55,11 @@ const (
 	ServerPSOutParams              uint16 = 0x1000
 )
 
+// HasCursorExistsFlag return true if cursor exists indicated by server status
```

Comment:
s/retrun/retruns
Add `.` at the end of a line. Please update other similar places.